src/main/resources/validationrules/civil/civil-admin-mapping.drl

```diff
@@ -418,3 +421,38 @@ then
       .build());
       logMsg("Rule : civil_case_allocator_admin_org_role");
 end;
+
+/*
+ * CIVIL admin and hearing centre team leader "case-allocator" Org role mapping.
```

Comment:
Can you adjust comment to remove new reference to the new roleId: e.g. `CIVIL admin "case-allocator" Org role mapping. v2.1.`

Reply:
Changes made as requested.

Reply:
Resolved in 80c7475aff3163939e73440c328bdf27bcdf563a.